src/integrationTest/java/org/opensearch/security/http/AuthInfo.java

```diff
@@ -18,13 +18,25 @@
 class AuthInfo {
 
     private final List<String> customAttributeNames;
+    private final boolean isInternal;
+    private final String authDomain;
 
-    @ConstructorProperties("custom_attribute_names")
-    public AuthInfo(List<String> customAttributeNames) {
+    @ConstructorProperties({ "custom_attribute_names", "user_is_internal", "user_auth_domain" })
```

Comment:
Maybe you have not gotten here yet, but my understanding is we are no longer going forward with these modifications correct? I think we are only supporting internal users meaning that there are no additional attributes required. 

If this is not the case let me know and I will provide a formal review for everything.

Reply:
@scrawfor99 See my question to Peter above. I believe we need to disambiguate the users in order to truly know that a user came from a particular auth domain. If there is a compelling reason not to move forward with this change then I will close, but that would mean that we'd be operating under the assumption that if we can lookup a username in the internal user list then assume its an internal user.

Consider the following case:

- `craig` is an internal user
- There is a SAML auth domain as well and another user called `craig` in the SAML backend

1. User authenticates as SAML `craig` and creates a detector
2. AD stores the detector and associates the detector with a user called `craig` - w/o linking this to the auth domain there is no way to know if this is `craig` the internal user or `craig` the SAML user
3. The security system permits the creation of this detector since it can find `craig` in the internal users even though its SAML `craig`.